assets/js/components/user-input/UserInputApp.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * User Input App.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default function UserInputApp() {
+	return featureFlags.widgets.userInput.enabled ? ( <div>TODO: Implement logic and UI.</div> ) : ( <div>You need a higher level of permission.</div> );
```

Comment:
Just to make this more readable and prepare for future expansion, it would be better to break this into multiple lines. I think we should have an if clause `if ( ! featureFlags.widgets.userInput.enabled )`, and in that return `__( 'Something went wrong.', 'google-site-kit' )`. This part will be removed once the feature goes to production. Then the TODO div can be the default return value.